docs/variants/dell_optiplex/releases.md

```diff
@@ -8,6 +8,31 @@ Dell OptiPlex 7010 SFF, DT and Dell OptiPlex 9010 SFF, MT.
 For details about our release process please read
 [Dasharo Standard Release Process](../../dev-proc/standard-release-process.md).
 
+## v0.1.0
+
+Test results for this release can be found
+[here](https://github.com/Dasharo/osfv-results/blob/main/boards/Dell/OptiPlex_7010_9010/results.csv).
+
+### Added
+
+- Support for Dell OptiPlex 7010/9010
+- [UEFI Boot Support](../../unified-test-documentation/dasharo-compatibility/30M-uefi-compatible-interface.md)
+- [Configurable boot order](../../unified-test-documentation/dasharo-compatibility/325-custom-boot-order.md)
+- Configurable boot options
+- [UEFI Secure Boot support](../../unified-test-documentation/dasharo-security/206-secure-boot.md)
+- [Custom boot logo](../../unified-test-documentation/dasharo-compatibility/304-custom-logo.md)
+- [Dasharo setup menu full screen mode support](https://github.com/Dasharo/dasharo-issues/issues/118)
+- [SMM BIOS write protection](../../dasharo-menu-docs/dasharo-system-features.md#dasharo-security-options)
+- [Firmware update mode](../../guides/firmware-update.md#firmware-update-mode)
+- [Setup menu password configuration](../../dasharo-menu-docs/overview.md#dasharo-menu-guides)
+- [USB stack disable option in setup menu](../../dasharo-menu-docs/dasharo-system-features.md#usb-configuration)
+- [Network stack disable option in setup menu](../../dasharo-menu-docs/dasharo-system-features.md#networking-options)
+- [Serial Console Redirection option](../../dasharo-menu-docs/dasharo-system-features.md#serial-port-configuration)
+
+### Known issues
+
+- [No ability to set SMM BIOS write protection](https://github.com/Dasharo/dasharo-issues/issues/971)
+
```

Comment:
The release notes are missing the SBOM section and information about where to find the binaries, similar to [MSI releases](https://docs.dasharo.com/variants/msi_z690/releases/#v112-2023-09-06:~:text=RTC%20on%20QubesOS-,Binaries,-This%20is%20a) as full binaries are available for subscribers only. 

> This is a Dasharo Pro Package Release. To obtain access to the pre-built binaries you will have to [become the Dasharo Pro Package subscriber. You will get the access to all of the firmware updates for the duration of the subscription via Dasharo Pro Package newsletter.

>To verify binary integrity with hash and signature please follow the instructions in [Dasharo release signature verification](https://docs.dasharo.com/guides/signature-verification) using [this key](https://raw.githubusercontent.com/3mdeb/3mdeb-secpack/master/dasharo/dell_optiplex_9010/dasharo-release-0.x-compatible-with-dell-optiplex-x010-signing-key.asc).